.env.example

```
@@ -14,8 +14,12 @@ SUPABASE_ANON_KEY=your_anon_key_here
 # Anitabi API (external data source, default is public endpoint)
 ANITABI_API_URL=https://api.anitabi.cn/bangumi
 
-# Google / Gemini — pydantic-ai LLM backend
+# Google / Gemini — primary pydantic-ai LLM backend
 GEMINI_API_KEY=your_gemini_api_key_here
+DEFAULT_AGENT_MODEL=google-gla:gemini-3.1-pro-preview
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DEFAULT_AGENT_MODEL key should go before the GEMINI_API_KEY key

(UnorderedKey)